server/routes/viewingIncidents/viewIncidents.ts

```diff
@@ -40,7 +40,7 @@ export default class ViewIncidentsRoutes {
     const submittedStatements = allStatements.filter(stmnt => stmnt.isSubmitted)
 
     if (tab === 'report') {
```

Comment:
As above, the call to the DB already returns edit history in descending chronological order so the the fix is to get the first element rather than the last element of the returned array.